Minor Touchup To Character Setup

Sun-Sam-San opened the pull request:
Reworks home systems, citizenships, factions, and religions to better lore consistency and improve quality of life.

New additions are in italics.

**Home System:** Tau Ceti, Sol, _Wintermute's Perches, Alpha Centauri, Trummer Flotilla, Xanu, Techno-Conglomerate Fleet, Scarab Fleet,_ New Ankara, Epsilon Eridani, X’yr Vharn’p, Jargon, S’rand’marr, Uueoa-Esa, Unset, Other
**Citizenship:** Republic of Biesel, Sol Alliance, _Frontier Alliance,_ Republic of Elyra, Eridani Federation, Empire of Dominia, Jargon Federation, People’s Republic of Adhomai, Izweski Hegemony, Zo’ra Hive, K’lax Hive, _C’thur Hive,_ None, Other
**Faction:** NanoTrasen, Hephaestus Industries, Idris Incorporated, Zeng-Hu Pharmaceuticals, Einstein Engines, Necropolis Industries, None, Other
**Religion Sure Looks Lacking:** Agnosticism, Deism, Christianity, Islam, Hinduism, Buddhism, _Folk Religion,_ Moroz Holy Tribunal, Qeblak, Weishii, S’rendarr & Messa, Ma’ta’ke, Th’akh, Sk'akh, _Aut'kah,_ None, Other

alsoandanswer commented:
!review

Sun-Sam-San commented:
!review

ben10083 commented:
How is a fleet a home system?

Sun-Sam-San commented:
> How is a fleet a home system?

you're born in space

Alberyk commented on code/modules/client/preferences_factions.dm:
```diff
@@ -13,58 +13,78 @@ var/global/list/seen_religions = list()
 	"S'rendarr & Messa",
 	"Ma'ta'ke",
-	"Th'akh",
-	"Sk'akh",
```
Sk'akh was removed by accident. Could you also add Aut'kah?

skull132 commented:
Time to be an opponent. Is there a point to adding the large myriad of contemporary religions to the list? There's a good few hundred of them, if you want to go deep enough. And what's worse, they bury the ones actually relevant to our lore. IMO just add new ones from the lore, instead of bringing in real life niche ones, like Jainism and Hoahaoism.

ben10083 commented:
I feel it is irrelevant to distern which fleet their """"home system""" is, I feel it would be better to just say "in space" or something along those linesz or just remove them.

OneOneThreeEight commented:
Agreeing with skull here. There's already an option to set your religion to anything you want, including Pastafarianism if you so wish but I'm sure that'd be moderated if it were detected.

Anyway, only the most notable factions/religions should be in the choices for religious faith. I do not think Caodaoism, Baha’i Faith, Tenrikyo, Jainism, Cheondoism, Hoahaoism have wiki pages detailing how they adapt to the Auroraverse, so what is the point of adding them to the list here?

VTCobaltblood commented on code/modules/client/preferences_factions.dm:
```diff
@@ -13,58 +13,84 @@ var/global/list/seen_religions = list()
 	"Th'akh",
 	"Sk'akh",
-	"Moroz Holy Tribunal"
+	"Aut'kah"
```
Aut'akh, not kah

Sun-Sam-San replied:
alright, thank you

VTCobaltblood commented on code/modules/client/preferences_factions.dm:
```diff
@@ -13,58 +13,84 @@ var/global/list/seen_religions = list()
-	"Gilthari Exports",
-	"Grayson Manufactories Ltd.",
-	"Aether Atmospherics",
-	"Zeng-Hu Pharmaceuticals",
```
Why was this removed?

VTCobaltblood commented on code/modules/client/preferences_factions.dm:
```diff
@@ -13,58 +13,84 @@ var/global/list/seen_religions = list()
 	"Zo'ra Hive",
 	"K'lax Hive",
-	"Empire of Dominia"
+	"C'thur Hive"
```
There is no C'thur citizenship. They're all citizens of Jargon Federation.

Sun-Sam-San replied:
thanks

Bygonehero commented:
Ingi Usang Entertainment Company,TakeOff! Entertainment,BP Entertainment Incorporated, Amijin-Kyodai Corporation are all from one planet and are relatively new to the lore. I get Venus is supposed to be the Hollywood of space but this many corporations from one place is a bit much. Lower it down to 1 or 2?

Sun-Sam-San commented:
> I feel it is irrelevant to distern which fleet their """"home system""" is, I feel it would be better to just say "in space" or something along those linesz or just remove them.

i don't understand why you think this if i'm honest

> Ingi Usang Entertainment Company,TakeOff! Entertainment,BP Entertainment Incorporated, Amijin-Kyodai Corporation are all from one planet and are relatively new to the lore. I get Venus is supposed to be the Hollywood of space but this many corporations from one place is a bit much. Lower it down to 1 or 2?

they're not all from venus. i'm going to get around to writing more stuff for them, but i planned to make them much more than just venusian corps. i was going to make amijin-kyodai the one in charge of kudan/ZunNet, for example

> Time to be an opponent. Is there a point to adding the large myriad of contemporary religions to the list? There's a good few hundred of them, if you want to go deep enough. And what's worse, they bury the ones actually relevant to our lore. IMO just add new ones from the lore, instead of bringing in real life niche ones, like Jainism and Hoahaoism.

> Anyway, only the most notable factions/religions should be in the choices for religious faith. I do not think Caodaoism, Baha’i Faith, Tenrikyo, Jainism, Cheondoism, Hoahaoism have wiki pages detailing how they adapt to the Auroraverse, so what is the point of adding them to the list here?

i went through and added all the religions with more than a million followers because i thought it'd be nice to have representation. i don't actually know anything about most of these religions if i'm honest. also, the current religion page is blank so i could add them to the wiki and they'd be just as developed, unless there's been a secret space catholic cabal i missed. anyways, here's the tables from wikipedia i used (i changed tenriism to tenrikyo to avoid confusion with tengriism)

![a](https://user-images.githubusercontent.com/43503975/58371857-021e7980-7edb-11e9-80d1-3ddb3d88a372.png)
![aa](https://user-images.githubusercontent.com/43503975/58371858-03e83d00-7edb-11e9-95b4-eef30739b667.png)

skull132 commented:
> i went through and added all the religions with more than a million followers because i thought it'd be nice to have representation. i don't actually know anything about most of these religions if i'm honest. also, the current religion page is blank so i could add them to the wiki and they'd be just as developed, unless there's been a secret space catholic cabal i missed. anyways, here's the tables from wikipedia i used (i changed tenriism to tenrikyo to avoid confusion with tengriism)

A million is a horribly small number when looking at a population of 7 billion. The point of all of these options is to bring lore to the forefront, not representation of current trends.

MarinaGryphon commented:
I think that the various forms of esoteric religion (i.e. paganism, Wicca, LaVeyan Satanism) would fit our lore much better.
Paganism and wicca are relevant in that they're very individualist, or at least non-centralised and personal. In an era dominated by megacorporations and authoritarian governments controlling almost every facet of your life, I can definitely see this appealing to people in Tau Ceti.
LaVeyan Satanism is also very might-makes-right, a sort of human counterpart to similar Unathi philosophy. It'd probably be appealing for similar reasons.
I also know several wiccan characters, and not having it as an option has always confused me.

Sun-Sam-San commented:
> A million is a horribly small number when looking at a population of 7 billion. The point of all of these options is to bring lore to the forefront, not representation of current trends.

>I think that the various forms of esoteric religion (i.e. paganism, Wicca, LaVeyan Satanism) would fit our lore much better.

i don't know how to make everyone happy, if i'm honest @skull132 @MarinaGryphon

ben10083 commented:
@Sun-Sam-San, I would start by removing most of the irrelevant religions with a population near 1 mil, as small religionz qould likely fade out if the trend of secularism carried onward to 2461, I feel only the bigger religions would still be standing. While it seems like a good idea to have as many choices as possible, it just causes intolerable levels of bloat.

skull132 commented:
> I think that the various forms of esoteric religion (i.e. paganism, Wicca, LaVeyan Satanism) would fit our lore much better.
> Paganism and wicca are relevant in that they're very individualist, or at least non-centralised and personal. In an era dominated by megacorporations and authoritarian governments controlling almost every facet of your life, I can definitely see this appealing to people in Tau Ceti.
> LaVeyan Satanism is also very might-makes-right, a sort of human counterpart to similar Unathi philosophy. It'd probably be appealing for similar reasons.
> I also know several wiccan characters, and not having it as an option has always confused me.

What fits our lore is irrelevant. What is relevant is **what is in our lore**. And the selection menus should do their finest to present that. For the rest, people can do their own research and use the "Other" setting.

Sun-Sam-San commented:
so what religions do i take out?

skull132 commented:
Literally all of the ones you added minus the akutha one.

Sun-Sam-San commented:
> Literally all of the ones you added minus the akutha one.

even taoism, shinto, and falun gong?

ParadoxSpace commented:
Shinto is certainly as relevant as Agnosticism, Deism, Christianity, Islam, Hinduism, Buddhism.

OneOneThreeEight commented:
> Shinto is certainly as relevant as Agnosticism, Deism, Christianity, Islam, Hinduism, Buddhism.

Yes, definitely. Nonetheless, what's in the lore now and is of mild importance is what should be considered of the highest priority in my opinion. I suggest anything above Judaism in population is what should be considered the most.

Bygonehero commented:
They are all from Venus. I checked.

Sun-Sam-San commented:
> Remove the new religions sans the ones actually relevant to our lore.

so every human religion because we've never actually expanded on religion lore at all or used it in any meaningful context?

skull132 commented:
> > Remove the new religions sans the ones actually relevant to our lore.
> 
> so every human religion because we've never actually expanded on religion lore at all or used it in any meaningful context?

To be quaint. Remove all _added_ religions sans Aut'akh.

Sun-Sam-San commented:
> > > Remove the new religions sans the ones actually relevant to our lore.
> > 
> > 
> > so every human religion because we've never actually expanded on religion lore at all or used it in any meaningful context?
> 
> To be quaint. Remove all _added_ religions sans Aut'akh.

i'm not wrong.
